styles/autocomplete.less

```diff
@@ -59,6 +59,7 @@ autocomplete-suggestion-list.select-list.popover-list {
     margin-top: 0;
     display: table;
     width: 100%;
+    margin-right: 10px; // Needed to prevent horizontal scrolling when right label is too long
```

Comment:
I can't reproduce the "horizontal scrolling" when removing the `margin-right: 10px;`.

It looks like the `10px` are for the scrollbar? Are you on OS X? I have the scrollbars only shown "when scrolling", but even when I turn them on permanently, it seems fine:

![autocomplete](https://cloud.githubusercontent.com/assets/378023/15951923/86dd289c-2ef7-11e6-8213-74c419852a48.gif)